introduction.md

```diff
@@ -0,0 +1,2 @@
+This is my introduction. I am Jose Mugaburu and seek to master github. I shall follow the ways of the Dojo on my path to mastery.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.